doc/Type/List.pod6

```diff
@@ -884,24 +884,40 @@ Defined as:
     multi method reduce(Any:D: &with)
     multi sub reduce (&with, +list)
 
-The first form is obviously a no-op. The second form generates a single
+The first form is obviously a no-op. The remaining forms generate a single
```

Comment:
Well, these methods are actually not defined in [List](https://github.com/rakudo/rakudo/blob/88d98225a4df61473ff5b935561719c56b04d1c2/src/core.c/List.pm6). Not in your PR, but as long as you're working with this, maybe the first could be eliminated, since it quite obviously does not belong here.